addons/stock/models/stock_config_settings.py

```diff
@@ -4,126 +4,79 @@
-    module_delivery_dhl = fields.Boolean("DHL integration")
-    module_delivery_fedex = fields.Boolean("Fedex integration")
-    module_delivery_ups = fields.Boolean("UPS integration")
-    module_delivery_usps = fields.Boolean("USPS integration")
-    # Warehouse and location usage_level
-    warehouse_and_location_usage_level = fields.Selection([
-        (0, 'Manage only 1 Warehouse with only 1 stock location'),
-        (1, 'Manage only 1 Warehouse, composed by several stock locations'),
-        (2, 'Manage several Warehouses, each one composed by several stock locations')
-        ], "Warehouses and Locations usage level")
+        help="Add and customize route operations to process product moves in your warehouse(s): e.g. unload > quality control > stock for incoming products, pick > pack > ship for outgoing products. \n You can also set putaway strategies on warehouse locations in order to send incoming products into specific child locations straight away (e.g. specific bins, racks).")
+    group_warning_stock = fields.Boolean("Warnings", implied_group='stock.group_warning_stock')
+    propagation_minimum_delta = fields.Integer(related='company_id.propagation_minimum_delta', string="No Rescheduling Propagation")
+    default_new_propagation_minimum_delta = fields.Boolean(string="No Rescheduling Propagation", default_model="stock.config.settings", help="Rescheduling applies to any chain of operations (e.g. Make To Order, Pick Pack Ship). In the case of MTO sales, a vendor delay (updated incoming date) impacts the expected delivery date to the customer. \n This option allows to not propagate the rescheduling if the change is not critical.")
+    module_stock_picking_wave = fields.Boolean("Picking Waves")
+    module_stock_calendar = fields.Boolean("Vendor Calendar for Reordering",
+        help='Scheduled dates of requests for quotation generated from reordering rules are based on vendor lead times (defined on products).\nBy default, such a scheduling is made on calendar days. This option allow to adapt the scheduling with vendor calendars to set on reordering rules.')
+    module_stock_barcode = fields.Boolean("Barcode Scanner")
+    module_delivery_dhl = fields.Boolean("DHL")
+    module_delivery_fedex = fields.Boolean("FedEx")
+    module_delivery_ups = fields.Boolean("UPS")
+    module_delivery_usps = fields.Boolean("USPS")
+    group_uom = fields.Boolean("Units of Measure",
+        implied_group='product.group_uom')
+    group_product_variant = fields.Boolean("Attributes & Variants",
+        implied_group='product.group_product_variant')
+    group_stock_wh_multi_locations = fields.Boolean('Storage Locations', implied_group='stock.group_stock_multi_locations', help="Store products in specific locations of your warehouse (e.g. bins, racks) and to track inventory accordingly.")
+    group_stock_several_wh_several_sl = fields.Boolean('Multi-Warehouses', implied_group='stock.group_stock_multi_warehouses')
```

Comment:
If we want more than one 'location' than enable 'group_stock_multi_locations',
If we want more than one 'warehouse' than enable 'group_stock_multi_warehouses' and
this two groups are different.